types/kendo-ui/index.d.ts

```diff
@@ -4046,7 +4046,7 @@ declare namespace kendo.ui {
         columnMenu?: boolean | GridColumnMenu;
         dataSource?: any|any|kendo.data.DataSource;
         detailTemplate?: string|Function;
-        editable?: boolean | GridEditable;
+        editable?: boolean | string | GridEditable;
```

Comment:
Might as well make it specifically `boolean | GridEditable | "inline" | "incell" | "popup"` to prevent other values from being accidentally used.